src/components/ServiceDetails/ServiceDetails.js

```diff
@@ -3,34 +3,57 @@ import ReactDOM from 'react-dom';
 import classNames from 'classnames';
 import styles from './ServiceDetails.css';
 import globalStyles from '../../styles/global.css';
+import {Card, CardHeader, CardText} from 'material-ui/Card';
 
 class ServiceDetails extends React.Component {
 	constructor() {
 		super();
 	}
- 
+
+  formatDate(date){
+	   return new Date(date).toLocaleDateString(); // Todo
+    };
 	render() {
-		if( this.props.data.length == 0 ){
+        const me =this;
```

Comment:
Is this properly indexed? Try to follow tabular indexing across the code.

Reply:
yes. It seems git issue

Reply:
Ok.